.github/workflows/release.yaml

```diff
@@ -99,25 +102,35 @@ jobs:
   windows-depends:
     strategy:
       matrix:
-        os: [windows]
+        os: [windows-2019, windows-2025]
```

Comment:
`windows` and `windows-arm64` are two GitHub actions runners available to Ollama. Changing this significantly affects the performance of the build. Please don't change it.

Reply:
Do you have acces windows arm runners or cross-compilation? It should comes in q2 2025… great news